src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -616,15 +622,21 @@ const renderSmallCollapse = (props, user, mediator, filtersAtDefault) => {
         <Row mb={2}>
           <Col xs={4}>
             <Text variant="xs" fontWeight="bold">
-              Artwork Info
+              {categoryText !== "Unknown" && mediumText !== "Unknown"
+                ? "Artwork Info"
+                : "Artwork Dimension"}
```

Comment:
Added the condition for the title as well. As if `categoryText` and `mediumText` both are unknown we will have only `dimension_text`